packages/react-component-library/src/components/TopLevelNavigation/Searchbar/Searchbar.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useRef, useState } from 'react'
 
 import { TextInput } from '../../TextInput'
-import { RightArrow } from '../../../icons'
+import { RightArrow, Search as SearchIcon } from '../../../icons'
```

Comment:
Have we got an issue to replace these old icons?

Reply:
Yes we have #1131